CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Added
 - Adds hdk access to keystore [#1148](https://github.com/holochain/holochain-rust/pull/1148)
 - Adds a `--path` option to `hc keygen` to specify the location of the generated keybundle. [#1194](https://github.com/holochain/holochain-rust/pull/1194)
+- Adds a `--quiet` option to `hc keygen` for machine-readable output, intended for use in scripts. [#1197](https://github.com/holochain/holochain-rust/pull/1197)
```

Comment:
This should additionally mention that this PR replaces `--silent` option, and also adds `--nullpass` option

Reply:
@Connoropolous disagree, because --silent and now --nullpass are temporary and meant to be undocumented

Reply:
--silent is simply renamed to --nullpass. --quiet is the new one.

Reply:
Ok. Can do